### PR TITLE
YLP-2031 migrate viz tests to dumb

### PR DESCRIPTION
**Pull request by TimCanu**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

File: packages/dumb/src/utils/datetime/datetime.util.ts [hunk 1]
```diff
@@ -129,66 +79,20 @@ export const getBrowserTimezone = (): string => {
  *
  * @return {String} timezoneName from timePrefs, browser, or fallback to 'UTC'
  */
-export const getTimezoneFromTimePrefs = (timePrefs: { timezoneAware: boolean, timezoneName: string }): string => {
+export const getTimezoneFromTimePrefs = (timePrefs?: { timezoneAware?: boolean, timezoneName?: string }): string => {
```
Comment (celia-rouis) [inline packages/dumb/src/utils/datetime/datetime.util.ts]: Would it be very complicated to reuse `TimePrefs` type here? You can even use `Partial<TimePrefs>` if you're not sure that any of the fields will be there

File: packages/dumb/src/utils/datetime/datetime.util.ts [hunk 2]
```diff
@@ -268,63 +172,11 @@ export const formatDuration = (duration: number, opts: { condensed: boolean }):
  *
  * @return {String} formatted datetime, e.g., 'Sunday, January 1'
  */
-export const formatLocalizedFromUTC = (utc: string | number | Date | moment.Moment, timePrefs: { timezoneAware: boolean, timezoneName: string }, format = getDayFormat()): string => {
+export const formatLocalizedFromUTC = (utc: string | number | Date | moment.Moment, timePrefs?: { timezoneAware?: boolean, timezoneName?: string }, format = getDayFormat()): string => {
```
Comment (celia-rouis) [inline packages/dumb/src/utils/datetime/datetime.util.ts]: Same question about using the `TimePrefs` type here

File: packages/dumb/src/beforeEach.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  ns: locales.namespaces,
+  defaultNS: locales.defaultNS,
+  fallbackNS: locales.fallbackNS,
+  resources: {
+    en: { yourloops: yourloopsEn, main: translationEn, params: parameterEn }
+  }
+}
+
+i18n.use(initReactI18next)
+i18n.init(i18nOptions).then(() => {
```
Comment (celia-rouis) [inline packages/dumb/src/beforeEach.js:53]: Is the `.then()` mandatory, given that it does nothing?

Comment (TimCanu) [reply packages/dumb/src/beforeEach.js:53]: It's not mandatory but the IDE gives me a warning if I don't have it

**Review by celia-rouis — APPROVED** [review]

**Pull request merged**